anonlink/similarities/__init__.py

```diff
@@ -5,13 +5,14 @@
 similarity of at least this threshold are returned. We call these the
 candidate pairs.
 
-Currently, the Dice Coefficient and Hamming Similarity are implemented.
-These work on binary strings. However, other similarity functions are
-possible as well.
+Currently, the Dice Coefficient and the Simple Matching Coefficient are
```

Comment:
Poor Sørensen always gets left out

Reply:
Diacritic make for bad Python identifiers 🤷🏼‍♂️